todo-app/src/TasksStore.js

```diff
@@ -1,52 +1,136 @@
-import {loadTasks, saveTasks} from './localStorageUtils';
-import {observable, computed, autorun, action} from 'mobx'
+//import {loadTasks, saveTasks} from './localStorageUtils';
+import {observable, computed , action , runInAction} from 'mobx';
+
+const api_url = "https://hyf-react-api.herokuapp.com";
 
 class TasksStore {
 
     @observable
-    tasks = loadTasks();
+    tasks = []
 
-    constructor() {
-        autorun(() => {
-            saveTasks(this.tasks);
-        });
-    }
+    @observable
+    loadingTasks = false 
 
     findTask(taskId) {
         return this.tasks.find(task => task.id === taskId)
     }
 
+    @action
+    loadTasks(showLoader = true){
+        if (showLoader) {
+            this.loadingTasks = true
+        }
+        fetch(`${api_url}/todos`)
+        .then(res =>{return res.json()})
```

Comment:
With arrow functions, you can actually skip the block declaration and `return` keyword if you're doing everything in one line:

    .then(res => res.json())

From the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions):

> Arrow functions can have either a "concise body" or the usual "block body".
    In a concise body, only an expression is specified, which becomes the explicit return value. In a block body, you must use an explicit return statement.